x-pack/plugin/runtime-fields/src/main/java/org/elasticsearch/xpack/runtimefields/fielddata/ScriptIpScriptDocValues.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+package org.elasticsearch.xpack.runtimefields.fielddata;
+
+import org.apache.lucene.document.InetAddressPoint;
+import org.elasticsearch.common.bytes.BytesArray;
+import org.elasticsearch.common.bytes.BytesReference;
+import org.elasticsearch.common.network.InetAddresses;
+import org.elasticsearch.index.fielddata.ScriptDocValues;
+import org.elasticsearch.xpack.runtimefields.IpScriptFieldScript;
+
+import java.io.IOException;
+import java.net.InetAddress;
+
+public class ScriptIpScriptDocValues extends ScriptDocValues<String> {
```

Comment:
double checking that this is the name you meant to use. I struggle to see what this class does

Reply:
It is a pretty terrible name! I talked with @javanna and it will think it over some more. Or more it so it is a bit more obvious.